server/chat-plugins/daily-spotlight.ts

```diff
@@ -128,7 +128,7 @@ export const pages: Chat.PageTable = {
 				const html = renderSpotlight(room.roomid, key, i);
 				buf += `<tr><td>${i || 'Current'}</td><td>${html}</td>`;
 				if (canManage) {
-					const deleteCommand = `/msgroom ${room.roomid},/removedaily ${key}, ${i}`;
+					const deleteCommand = `/msgroom ${room.roomid},/removedaily ${key}${i === 0 ? '' : `, ${i}`}`;
```

Comment:
```suggestion
					const deleteCommand = `/msgroom ${room.roomid},/removedaily ${key}, ${i}`;
```

If you fixed `, 0`, you shouldn't need this code either.

Reply:
part of the fix with 0 is using '/removedaily key' instead of '/removedaily key, 0' because doing the latter doesn't actually fully remove the daily it just removes the 0th key which is why it still shows up in the menu. This code is needed in order to account for 0 not wanting it's number to be put as a que in /removedaily.